DAL/Migrations/201803130904498_InitialMigration.cs

```diff
@@ -0,0 +1,28 @@
+namespace iTechart.Survey.DAL.Migrations
+{
+    using System;
```

Comment:
Unused reference. Using should be at the top of file, outside namespace. Remove empty line in the end of file